topics/unreal-engine/index.md

```diff
@@ -0,0 +1,12 @@
+---
+aliases: ue4, unrealengine, unrealengine4, unreal-engine-4
+created_by: Epic Games
+display_name: Unreal Engine
+github_url: https://github.com/EpicGames/UnrealEngine
+logo: unreal-engine.png
+short_description: Developers choose Unreal Engine to create awesome games, and experiences for PC, mobile, console, VR, and AR.
```

Comment:
I think the comma before "and experiences" is unnecessary.

Reply:
Agreed. I was terrified that Oxford comma test would be out to get me :finnadie: